DOCUMENTATION.md

```diff
@@ -18,19 +19,19 @@ Geocint consists of 3 different parts:
 - [geocint-runner](https://github.com/konturio/geocint-runner) - a core part of the pipeline, includes utilities and initial Makefile
 - [geocint-openstreetmap](https://github.com/konturio/geocint-openstreetmap) - a chain of targets for downloading, updating and uploading
 to database OpenStreetMap planet dump
-- [geocint-private] any repository that contains your additional functionality
+- [geocint-custom] any repository that contains your additional functionality
 
-During the installation process, you should clone all these repositories to ~/.
+During the installation process, you should clone all these repositories to working directory.
 
-During the installation of the geocint pipeline the next folders will be created in your home directory (~/):
+During the installation of the geocint pipeline the next folder will be created in your working directory:
 
 - [public_html] - any public html that you want to share
```

Comment:
what if I don't want to share them?
Any other files, like make_profile.db? I saw it later on Also when running... section, but it might be united

Reply:
Well, it is assume that geocint has both backend and frontend parts (web-dashboard for front end), and it should be documented in the user manual.

If you do not want to share  dashboard, logs,  etc, you can switch web-server manually (or just empty the public_html directory)